src/alpine/3.17/helix/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apk add --upgrade --no-cache \
         openssl-dev \
         perl
 
-RUN git clone --depth 1 --single-branch --branch main --recursive https://github.com/microsoft/msquic /tmp/msquic
+RUN git clone --depth 1 --single-branch --branch v2.4.1 --recursive https://github.com/microsoft/msquic /tmp/msquic
```

Comment:
This pins to a specific version while the other Dockerfiles are essentially getting the latest available version. Pinning causes maintenance issues. It would be ideal if the Alpine Dockerfiles could have the same behavior. Consider querying the GitHub releases API to determine the latest release and targeting that branch instead. Example of such a pattern: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/75019f244373eeac6238e76dc72ed751a4f9f173/src/nanoserver/1809/helix/amd64/Dockerfile#L6-L8

Reply:
It makes sense, but I'll probably do this as a follow-up PR.

Reply:
> It makes sense, but I'll probably do this as a follow-up PR.

That's fine.

> Ah, this might not work in this case, since we're not always using latest release on GitHub, we're actually using latest pre-release build.

That information should still be available from the API to make whatever choice you want.